subsys/metrics/src/main/conf/conf.d/metrics.conf

```diff
@@ -0,0 +1,9 @@
+[metrics]
+org.commonjava.indy.metrics.isenable=true
```

Comment:
Sorry to be nitpicky, but can you change this to `enabled` and the next line to `reporter`? You'll have to update default-metrics.conf and the IndyMetricsConfig constants, but this will make it more consistent with the rest of the Indy configuration.